File: examples/Fibo.java
  public class Fibo {
  public static void main(String[] args) {
    int i = 0;
    int a[] = new int[10];
    a[0] = 1;
    a[1] = 1;
    for (i = 2; i < 10; i++) {
      a[i] = a[i - 1] + a[i - 2];
    };
  }
}

File: examples/fibo.clj
(defn fibo
  ([] (fibo 1 1))
  ([s1 s2] (lazy-seq (cons s1 (fibo s2 (+ s1 s2))))))

(take 5 (fibo))

(take-while (partial > 50) (fibo))

(take 5 (take-nth 10 (fibo)))
